### ISSUE TITLE
Black Tags displayed as white (with white text)

### ISSUE THREAD

**Issue by robertpeteuil** [inkdropapp/vibrant-dark-ui#7]
Just noticed that black tags are displaying as white with white text.  It appears to be caused by this theme, and I tested it on two different computers.

I found this [old issue and resolution](https://forum.inkdrop.app/t/choice-of-colour-of-tag-causes-it-to-not-display-well/1400/2), and the fix still works.

The fix is to use custom the customer css

```less
.ui.black.label, .ui.black.labels .label {
    background-color: #1e2225!important;
    border-color: #1e2225!important;
    color: #f9fafb!important;
}
```

**Comment by craftzdog** [comment]
Hi Robert,

Thanks for the report. Fixed it!

**Comment by craftzdog** [comment]
Landed in [v1.1.1](https://github.com/inkdropapp/vibrant-dark-ui/releases/tag/v1.1.1)🎉

**Comment by robertpeteuil** [comment]
Excellent, thanks as always!

Must be that fancy new MacPro that's allowing you to get these fixes out so quickly.
